proto/proto/internal/store.proto

```diff
@@ -0,0 +1,342 @@
+// Unified specification of the internal store gRPC APIs.
```

Comment:
Maybe we should a have small readme here `proto/proto/README.md` explaining the rationale?